packages/blade/src/components/Icons/Icons.stories.tsx

```diff
@@ -67,15 +69,38 @@ export default {
   },
 } as Meta<IconProps>;
 
-const IconTemplate: ComponentStory<ComponentType<IconProps & { icon: string }>> = ({
+const IconTemplate: ComponentStory<ComponentType<IconProps & { icon?: string }>> = ({
   icon,
   ...args
 }) => {
-  const IconComponent = iconMap[icon];
-  return <IconComponent {...args} />;
+  if (icon) {
+    const IconComponent = iconMap[icon];
+    return <IconComponent {...args} />;
+  }
+  return (
```

Comment:
We don't have to add this in same template, We can create a new export where we loop over iconMap and render icons and we can let this icon template be as it is maybe?

You can check out how we do it in `Badge.stories.tsx`

Reply:
What should be the name of the story(s) that includes all the icons?

Reply:
How about `All Icons` 🤔 ?

Reply:
Yup, this is exactly what I had in mind. But none of the other components had a `All ~` story so I skipped over it.